File: Lab5/player/src/main/java/com/example/demo/DTO/PatchPlayerRequest.java
package com.example.demo.DTO;

import lombok.*;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor(access = AccessLevel.PRIVATE)
@ToString
@EqualsAndHashCode
public class PatchPlayerRequest {
    private Long id;
    private String name;
    private Long clubId;
    private int overall;
}

File: lab6/angularclient/src/app/patch-player-request.ts
export class PatchPlayerRequest {
    id: number;
    name: string;
    clubId: number;
    overall: number;

    constructor(id: number, name: string, clubId: number,overall: number) {
        this.id = id;
        this.name = name;
        this.clubId = clubId;
        this.overall = overall;
    }
}
